File: Data Structures & Algorithms/jump-game-ii/submission-1.cpp
class Solution {
public:
    int jump(vector<int>& nums) {
        int l = 0, r = 0;
        int count = 0;
        while(l < nums.size() - 1 && r < nums.size() - 1){
            int mm = l;
            for(int i = l;i<=r;i++){
                mm = max(i + nums[i], mm);
            }
            l = r + 1, r = mm;
            count += 1;
        }
        return count;
    }
};

File: Data Structures & Algorithms/non-overlapping-intervals/submission-1.py
class Solution:
    def eraseOverlapIntervals(self, intervals: List[List[int]]) -> int:
        intervals.sort()
        last = -100000
        count = 0
        for interval in intervals:
            if interval[0] < last:
                last = min(last, interval[1])
                count += 1
            else:
                last = interval[1]
        return count